contents/docs/error-tracking/_snippets/cli/inject.mdx

```diff
@@ -1,4 +1,6 @@
 ```bash
-# Inject metadata in files to resolve errors
-posthog-cli sourcemap inject --directory ./path/to/assets
+# Inject release and chunk metadata into sourcemaps
+posthog-cli sourcemap inject --directory ./path/to/assets --project my-app --version 1.2.3
 ```
+
```

Comment:
Most places where we'll use this will have this step: https://posthog.com/docs/error-tracking/upload-source-maps/nuxt#authenticate-the-posthog-cli

Maybe we don't need this disclaimer about authenticating?